tags/tag_set_codec.go

```diff
@@ -159,8 +159,8 @@ func (eg *encoderGRPC) bytes() []byte {
 	return eg.buf[:eg.writeIdx]
 }
 
-// EncodeToFullSignature will encode the tagSet to []byte.
-func EncodeToFullSignature(ts *TagSet) []byte {
+// Encode will encode the tag set into a []byte.
```

Comment:
Let's make the comment helpful (we know it takes a tag set and returns a []byte already); something like "Encode the tag set into the grpc wire format(?)."

BTW, I don't know what this tags package has to do with grpc.

Reply:
I didn't want to give specific examples because this is a generic binary encoder/decoder. gRPC ends up using the binary encoding is an implementation detail for the gRPC propagation strategy.

Reply:
What I meant is the comment in the current form doesn't add any value and should explain more about what the output is for. I also wanted to highlight that the appearance of grpc in this package doesn't look right (and it seems that succeeded. :-))

Reply:
Done

Explained further that it is useful to propagate the tag sets on write in binary format.